content/blog/2020-announce-devsprints.md

```diff
@@ -0,0 +1,30 @@
+Title: git commit -s -m "Announing Devsprints 2020"
+Date: 2020-09-16 00:00
+Author: 
+Category: 2020
+Slug: announce-devsprints-2020
+Summary: Announcing the PyCon India 2020 devsprints 
+
+
+PyCon India 2020 is THE place for developers from across the country and beyond to gather and celebrate Python. But what good is a PyCon without getting your hands dirty writing some code? Hence, for all open-source enthusiasts out there, the PyCon India team is happy to announce Devsprints @ PyCon India 2020.
+
+## What is a Devsprint?
+
+During **Devsprints**, mentors/contributors of popular OSS projects guide participants through the end-to-end process of an OSS contribution. Right from setting up the build environments, to developing and testing code, to submitting patches for bugs and features. If you are stuck anywhere, mentors will be available right on the stream to help you figure out issues and resolve them as best as they can! From past experiences, we’ve seen a lot of people start off contributing to FOSS through PyCon India Devsprints and later go on to become regular contributors to various projects across the web.
+
+## When and Where? Are Devsprints really free?
+
+Devsprints will be conducted online on 4th and 5th October 2020 on the Hopin.to conference platform. The [Devsprints tickets](https://in.pycon.org/2020/#ticket) are free of charge and limited in number as slots for the devsprint are limited. Please make sure to grab yours when they are released if you are looking to kickoff your FOSS journey right!
```

Comment:
Tickets are gone now. I don't know if there is a point of that.

Reply:
More tickets will come. right?

Reply:
We don't have a guarantee on that. :(

Reply:
I believe we will have more. Sanchit said. We also announced at Flames event that there'll be more. And also promising the project maintainers while getting more projects.

Reply:
All I am saying is we don't need to put speculation into the words of the blog. This thing needs to be communicated through social media.

Reply:
Makes sense. 👍🏼